v0/destinations/wootric/util.js

```diff
@@ -216,106 +247,50 @@ const createDeclinePayloadBuilder = message => {
  * @param {*} destKey
  */
 const flattenProperties = (payload, destKey) => {
-  if (payload.properties) {
-    let rawProperties = {};
+  const rawProperties = {};
+  if (isDefinedAndNotNullAndNotEmpty(payload.properties)) {
     Object.entries(payload.properties).forEach(([key, value]) => {
       rawProperties[`${destKey}[${key}]`] = `${value}`;
     });
-    return { ...rawProperties };
   }
+  return rawProperties;
 };
 
 /**
  * Formats identify payload
- * ref: https://docs.wootric.com/api/#get-a-specific-end-user-by-phone-number
  * @param {*} payload
  */
 const formatIdentifyPayload = payload => {
-  if (payload.last_surveyed) {
-    payload.last_surveyed = `${payload.last_surveyed}`;
+  const rawPayload = { ...payload };
```

Comment:
Lets add bit more contextual documentation for this function. Also the this function is stringifying dates but not formatting anything else for the entire payload. Contextual naming would be helpful?

Reply:
updated